2021/Firera/sysMonitor_PC/conf.py

```diff
@@ -0,0 +1,7 @@
+# CONFIGURATION FILE (CHANGE WITH YOUR INFLUXDB DATA)
+
+INFLUXDB_TOKEN = 'Sjw-tnzb3mKiRoQMfSEi_3pjT7Q2ewL3SUSNStqBJX0WtjP5RsPuLXl75iJXIrVQ7FYX_RijAGI2zxhslPKtQA=='
```

Comment:
Togli per favore questo valore anche se capisco ci sono poche probabilità che ce ne siano due identici? Ogni credenziale non deve essere committata in git